src/components/atoms/DarkModeButton/index.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+
+import React, { useEffect } from 'react'
+import Image from 'next/image'
+import Assets from '@/config/assets'
+import useStorage from '@/hooks/useStorage'
+
+export default function DarkModeButton() {
+  const [darkMode, setDarkMode] = useStorage<boolean>({
+    storageType: 'local',
+    key: 'pcc-darkmode',
+    initialValue: window.matchMedia('(prefers-color-scheme: dark)').matches,
+  })
+
+  useEffect(() => {
+    if (darkMode) {
+      document.body.classList.add('pcc-theme--dark')
+      document.body.classList.remove('pcc-theme--light')
+    }
+  }, [darkMode])
+
+  const handleDarkmodeClick = () => {
+    document.body.classList.toggle('pcc-theme--light')
+    document.body.classList.toggle('pcc-theme--dark')
+    setDarkMode(!darkMode)
+  }
+
+  return (
+    <button onClick={handleDarkmodeClick}>
+      {darkMode ? (
+        <Image
+          src={Assets.LIGHTMODE_SVG_PATH}
+          width={30}
+          height={30}
+          alt="dark mode button"
+        />
+      ) : (
+        <Image
+          src={Assets.DARKMODE_SVG_PATH}
+          width={30}
+          height={30}
+          alt="dark mode button"
+        />
+      )}
```

Comment:
```suggestion
        <Image
          src={darkMode ? Assets.LIGHTMODE_SVG_PATH : Assets.DARKMODE_SVG_PATH}
          width={30}
          height={30}
          alt="dark mode button"
        />
      )}
```

요런 식도 고려해 볼 수 있을 것 같아요!

Reply:
오.... 좋습니다 👍